templates/github/.github/workflows/nightly.yml.j2

```diff
@@ -30,6 +30,9 @@ jobs:
           {%- if docs_test %}
           - TEST: docs
           {%- endif %}
+          {%- if test_azure %}
+          - TEST: azure
+          {%- endif %}
```

Comment:
I've been thinking about only enabling it at nightly job

Reply:
I would say, new tests should run more often. What they are supposed to test is probably a hot area of code change. Once you are confident about them, you can move them to nightly only.

Reply:
I was worried about the CI load, but I agree with you